src_nbgl/ui_settings.c

```diff
@@ -34,19 +34,25 @@ static bool navCallback(uint8_t page, nbgl_pageContent_t* content) {
             switches[index++] =
                 (nbgl_layoutSwitch_t){.initState = N_storage.dataAllowed ? ON_STATE : OFF_STATE,
                                       .text = "Blind signing",
-                                      .subText = "Enable transaction blind\nsigning",
+                                      .subText =
+                                            "Enable transaction blind\n"
```

Comment:
Why do you need to split thetext like this?

Reply:
Updated the codespell version to avoid this workaround